mindmeld/components/dialogue.py

```diff
@@ -906,6 +909,18 @@ def _end_slot_fill(self, request, responder, async_mode):
             params=request.params,
         )
 
+        if self._invoked:
+            kwargs = {'domain': request.domain, 'intent': request.intent}
```

Comment:
We should add the full list of args here:
https://github.com/cisco/mindmeld/blob/master/mindmeld/components/dialogue.py#L104

```
        key_kwargs = (
            ("domain",),
            ("intent",),
            ("has_entity", "has_entities"),
            ("targeted_only",),
            ("default",),
        )

Reply:
Unless we have a conscious decision only to use domain & intent for mapping

Reply:
Hmm nvm, I think I misread this section, you are mapping the domain and intent to the automatic slot filling handler itself.

Reply:
I think we only need to set `targeted_only` to True and set the target dialogue state.

Reply:
thanks for the help! 😄